packages/fastify-renderer/src/node/Plugin.ts

```diff
@@ -77,7 +77,12 @@ export class FastifyRendererPlugin {
    * Implements the backend integration logic for vite -- pulls out the chain of imported modules from the vite manifest and generates <script/> or <link/> tags to source the assets in the browser.
    **/
   pushImportTagsFromManifest = (bus: RenderBus, entryName: string, root = true) => {
-    const manifestEntry = this.clientManifest![entryName]
+    let manifestEntry = this.clientManifest![entryName]
```

Comment:
This _might_ be something to re-visit. I don't understand the manifest JSON format 100%, so I'm not sure how best to extract the correct name.

This wasn't working properly previously.

Reply:
This is one of my older changes so I hope I'm not mixing anything up here.

Basically, the way that the imperative routes are structured in the json does not line up with how they are looked-up.

This change fixes it, but it is moot given that imperative routes don't work in prod anyways.

Reply:
I added a comment to that effect. Won't be addressing it in this PR.